<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BatteryChargerCalibrateDialogue</class>
 <widget class="QDialog" name="BatteryChargerCalibrateDialogue">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>302</width>
    <height>369</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>300</y>
     <width>92</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="firstValueLineEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="secondValueLineEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="firstValueSetButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>100</y>
     <width>92</width>
     <height>28</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Enter the first measured voltage</string>
   </property>
   <property name="text">
    <string>Set First</string>
   </property>
  </widget>
  <widget class="QPushButton" name="secondValueSetButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>140</y>
     <width>92</width>
     <height>28</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Enter the second measured voltage</string>
   </property>
   <property name="text">
    <string>Set Second</string>
   </property>
  </widget>
  <widget class="QLabel" name="minimumValueLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>260</y>
     <width>101</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Minimum Value</string>
   </property>
  </widget>
  <widget class="QLabel" name="maximumValueLabel">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>260</y>
     <width>111</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Maximum Value</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="minimumLcdNumber">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>220</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="segmentStyle">
    <enum>QLCDNumber::Flat</enum>
   </property>
  </widget>
  <widget class="QLCDNumber" name="maximumLcdNumber">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>220</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="midLineWidth">
    <number>0</number>
   </property>
   <property name="segmentStyle">
    <enum>QLCDNumber::Flat</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="commitButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>300</y>
     <width>92</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Commit</string>
   </property>
  </widget>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>231</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Channel Calibration</string>
   </property>
  </widget>
  <widget class="QLabel" name="subTitle">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>231</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="clearEntriesButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>180</y>
     <width>92</width>
     <height>28</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Enter the second measured voltage</string>
   </property>
   <property name="text">
    <string>Clear Entries</string>
   </property>
  </widget>
  <widget class="QLabel" name="errorLabel">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>340</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>firstValueLineEdit</tabstop>
  <tabstop>firstValueSetButton</tabstop>
  <tabstop>secondValueLineEdit</tabstop>
  <tabstop>secondValueSetButton</tabstop>
  <tabstop>clearEntriesButton</tabstop>
  <tabstop>commitButton</tabstop>
  <tabstop>closeButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
